src/filters/util.js

```diff
@@ -3,26 +3,18 @@
 angular.module('openshiftCommonUI')
   .filter("toArray", function() {
     return function (items) {
```

Comment:
It does work but I thought it was a bit odd to use it this way. I will update it since it's probably just me.